Source/OxyPlot/Rendering/OxyColor.cs

```diff
@@ -216,7 +192,7 @@ public static OxyColor FromHsv(double[] hsv)
         public static OxyColor FromHsv(double hue, double sat, double val)
         {
             double g, b;
-            double r = g = b = 0;
+            var r = g = b = 0;
```

Comment:
This is hideous, though not all `var`'s fault. Should change these to straight declarations, and change the switch further down to throw for unexpected values of `i`.

Reply:
Changed these declarations for something more sensible